content/200-concepts/100-components/01-prisma-schema/07-names-in-underlying-database.mdx

```diff
@@ -184,3 +186,34 @@ The following table lists the name of each constraint and index in the underlyin
 | `@@index` (on `Post`)              | No                 | `My_Custom_Index_Name`               |
 | `@id` (on `Post` > `id` field)     | Yes                | `Post_pk`                            |
 | `@relation` (on `Post` > `author`) | Yes                | `Post_authorName_fkey`               |
+
+### Related: Naming indexes and primary keys for Prisma Client
+
+Additionally to `map`, the `@@id` and `@@unique` attributes take an optional `name` argument that allows you to customize your Prisma client API.
+
+On a model like:
+
+```prisma
+model User {
+  firstName String
+  lastName  String
+
+  @@id([firstName, lastName])
+}
+```
+
+the default API for selecting on that primary key uses a generated combination of the fields:
+
+```ts
+prisma.user.findUnique({
+  where: { firstName_lastName: { firstName: 'Paul', lastName: 'Panther' } },
+})
+```
+
+Specifying `@@id([firstName, lastName], name: "fullName")` will change the Prisma Client API to this instead:
+
+```ts
+prisma.user.findUnique({
+  where: { fullName: { firstName: 'Paul', lastName: 'Panther' } },
+})
+```
```

Comment:
Moved this section to the bottom as I think it is not _really_ about "Names in the underlying database" but certainly related and relevant. Also changed the order of the description to go from default to customized and actually show the model this changes.